pkg/services/ngalert/schedule/schedule.go

```diff
@@ -328,6 +328,7 @@ func (sch *schedule) ruleRoutine(grafanaCtx context.Context, key ngmodels.AlertR
 		start := sch.clock.Now()
 
 		schedulerUser := &user.SignedInUser{
+			// FIXME: add is service account and refactor to a service account instead of a user
```

Comment:
@grafana/grafana-edge-squad wanted to add you into the loop to know how to account for grafana service accounts from the signedinuser and that we have added the `IsServiceAccount` attribute to the SignedInUser.